### ISSUE TITLE
Hello is api.betabotz.eu.org is down?

### ISSUE THREAD

**Issue by Masterolic** [ERLANRAHMAT/betabotz-tools#2]
I think the domain is expired dns resolve failed

**Comment by ERLANRAHMAT** [comment]
Yes, I will fix it as soon as possible because there is a problem, please contact me via WhatsApp +62 895-6281-17900

**Comment by Masterolic** [comment]
> Yes, I will fix it as soon as possible because there is a problem, please contact me via WhatsApp +62 895-6281-17900

Hello is it still available

**Comment by ERLANRAHMAT** [comment]
contact me on whatsapp
